src/app/static/src/app/components/adr/SelectRelease.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <div id="selectRelease"></div>
+</template>
+<script lang="ts">
+    import Vue from "vue";
+    import { mapActionByName } from "../../utils";
+
+    export default Vue.extend({
+        props: ["selectedDataset"],
+        methods: {
+            getReleases: mapActionByName("adr", "getReleases"),
+        },
+        watch: {
+            selectedDataset(id) {
+                this.getReleases(id);
```

Comment:
I think there's a bug here - the perils of incomplete UI having unintended consequences!

If I click the cross in the datasets dropdown then I now get an ADR error displayed in the UI, because `id` is `undefined` here

Reply:
I've put a check in the watcher that id is truthy